src/earn/EarnInfoScreen.tsx

```diff
@@ -78,13 +77,13 @@ export default function EarnInfoScreen() {
         <View style={styles.detailsContainer}>
           <DetailsItem
             icon={<EarnCoins size={ICON_SIZE} color={Colors.black} />}
-            title={t('earnFlow.earnInfo.details.earn.title')}
-            subtitle={t('earnFlow.earnInfo.details.earn.subtitle')}
+            title={t('earnFlow.earnInfo.details.work.title')}
+            subtitle={t('earnFlow.earnInfo.details.work.subtitle')}
           />
           <DetailsItem
-            icon={<Logo size={ICON_SIZE} color={Colors.black} />}
-            title={t('earnFlow.earnInfo.details.manage.titleV1_92', { appName: APP_NAME })}
-            subtitle={t('earnFlow.earnInfo.details.manage.subtitleV1_92', { appName: APP_NAME })}
+            icon={<Manage size={ICON_SIZE} color={Colors.black} />}
```

Comment:
should this and coins icon be 20 per the design?

Reply:
updated in [cd2b56a](https://github.com/valora-inc/wallet/pull/5977/commits/cd2b56acba08660492020bb1b5764cfc924a171f)

Yes, looks like Manage and EarnCoins should be 20. Also looks like the background circle size should be 32. Updated both.